overlay.nix

```diff
@@ -108,8 +85,6 @@ in rec {
 
   evmdis = self.callPackage ./nix/evmdis.nix {};
 
-  token = self.callPackage (import ./src/token) {};
```

Comment:
I think this may have some users still, not sure.

Reply:
Could just delete and see if there are complaints.

Reply:
We should keep this.